year-pages/2020.md

```diff
@@ -2,6 +2,7 @@
 
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
+| [Distributed Systems Conf](https://dsconf.in/) | 01 Feb | Hotel Novotel, Viman Nagar, Pune | India’s foremost conference on Distributed Systems | No |
```

Comment:
Please sort this in reverse chronological order as per date :)

Reply:
Ohh. I thoughts it is ordered by conference title. Will do, thanks!